model/ExpandedLicensing/Vocabularies/NoAssertionLicense.md

```diff
@@ -0,0 +1,24 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# NoAssertionLicense
+
+## Summary
+An Individual Value for License when no assertion can be made about it's actual value.
+
+
+## Description
+
+NoAssertion should be used if the SPDX creator has attempted to but cannot reach a reasonable objective determination;
+the SPDX creator has made no attempt to determine this field; or
```

Comment:
as discussed in https://github.com/spdx/spdx-3-model/issues/289, compared to SPDX2 we have also `null` (or not including any licensing information) as an value and it should be clearly distinct from `NoAssertion`. I think I understood that `NoAssertion` should only be valid if and only if "the SPDX creator has intentionally provided no information".

I think we need to align this with https://github.com/spdx/spdx-3-model/issues/289#issuecomment-1589450596

Reply:
The comment implies changing the definitions for the properties, not the NoneLicense and NoAssertionLicense.

@maxhbr if you agree, I can add a suggested text change to the properties along with this PR.

@kestewart - see the comments in issue #289 for more context

Reply:
> @maxhbr if you agree, I can add a suggested text change to the properties along with this PR.

yes, that would match what was discussed (although I am still not happy with it).


Since the 2.3 Noassertion now maps to two values (not present and noassertion in 3.0), we need to do a choice regarding migration. Maybe that should also be added to the description as a comment.

Reply:
@maxhbr @kestewart I added proposed text to concludedLicense.md and declaredLicense.md - please review